skimage/measure/_regionprops.py

```diff
@@ -1253,14 +1274,14 @@ def regionprops(label_image, intensity_image=None, cache=True,
     if not np.issubdtype(label_image.dtype, np.integer):
         if np.issubdtype(label_image.dtype, bool):
             raise TypeError(
-                    'Non-integer image types are ambiguous: '
-                    'use skimage.measure.label to label the connected'
-                    'components of label_image,'
-                    'or label_image.astype(np.uint8) to interpret'
-                    'the True values as a single label.')
+                'Non-integer image types are ambiguous: '
+                'use skimage.measure.label to label the connected'
+                'components of label_image,'
+                'or label_image.astype(np.uint8) to interpret'
+                'the True values as a single label.')
         else:
             raise TypeError(
-                    'Non-integer label_image types are ambiguous')
+                'Non-integer label_image types are ambiguous')
```

Comment:
Do you mind reverting unrelated formatting changes? They will make git-blame harder to follow.

Reply:
I hope I caught all of them :innocent: